pkg/table/table.go

```diff
@@ -260,6 +287,16 @@ func (m *Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		case m.ShowHelp && key.Matches(msg, m.KeyMap.CloseFullHelp):
 			m.ShowHelp = false
 			return m, nil
+		case key.Matches(msg, m.KeyMap.Filter):
+			m.filterInputEnabled = !m.filterInputEnabled
+			m.updateFilteredRows()
+			m.updatePagination()
+			m.updateColumnWidths()
+			cmd := tea.Cmd(nil)
```

Comment:
nitpick: I prefer using the `var` syntax when leaving the variable to default value

```suggestion
			var cmd tea.Cmd
```